cot/src/session/store/redis.rs

```diff
@@ -140,6 +142,7 @@ impl RedisStore {
     }
 }
 
+#[allow(clippy::cast_sign_loss)]
```

Comment:
```suggestion
#[expect(clippy::cast_sign_loss)]
```

`expect` will throw an error when the lint will no longer be triggered for whatever reason (so that we instantly know we should remove the `expect`)